src/api/controllers/authentication.js

```diff
@@ -41,7 +41,10 @@ function register(req, res) {
                     user = new Recruiter();
                     break;
                 case 'admin':
-                    user = new Admin();
+                    return res.status(403).json({
```

Comment:
so how do we create the admin account then?

Reply:
mongo shell

Reply:
idk, maybe we don't need to restrict it entirely, but we need some measure in place against people registering as admin. Right now it's easier to register as an admin than as a recruiter, at least via the api